simulators/nav/src/components/NavSimulator.vue

```diff
@@ -355,6 +358,8 @@ export default class NavSimulator extends Vue {
 
   /* Publish the given LCM message on the given channel. */
   private publish(channel:string, payload:any, odomOrObstacle:boolean):void {
+
```

Comment:
`simulators/nav` won't compile unless you delete this blank line on 361 (linting issue) when I tried to run it